client/src/pages/CreateSurvey/Options/Options.jsx

```diff
@@ -35,17 +60,34 @@ const Option = ({ optionIndex, questionIndex }) => {
     payload.optionIndex = optionIndex;
 
     dispatch({ type: 'MOVE_OPTION_UP', payload });
+    dispatch({
+      type: 'SWAP_ERRORS',
+      payload: [
+        `${questionIndex}${optionIndex}`,
+        `${questionIndex}${optionIndex - 1}`,
+      ],
+    });
   };
 
   const handleDownClick = () => {
     const payload = {};
     payload.questionIndex = questionIndex;
     payload.optionIndex = optionIndex;
     dispatch({ type: 'MOVE_OPTION_DOWN', payload });
+    dispatch({
+      type: 'SWAP_ERRORS',
+      payload: [
+        `${questionIndex}${optionIndex}`,
+        `${questionIndex}${optionIndex + 1}`,
+      ],
+    });
   };
 
   const isNotFirstQuestion = optionIndex !== 0;
   const isNotLastQuestion = options.length - 1 !== optionIndex;
+  const optionTextIsBiggerThanMaxValue = text && text.length > 280;
```

Comment:
This variable name is being used above also, but within a function definition.

Reply:
Yea true, but this one listens to the value `text` but the other one listens for the events value